build/ChoosePowerShell.cmd

```diff
@@ -0,0 +1,23 @@
+:: where.exe does not exist in windows container, application specific test must be used to check for existence
+
+PowerShell -Command Write-Host "a"
+
+IF %ERRORLEVEL% == 0 (
+
+    set PowerShell=PowerShell
+
+    exit /B 0
+)
+
+pwsh -Command Write-Host "a"
```

Comment:
I believe we are printing `a` only to validate if `pwsh` is a valid command in `PATH`. How about using `WHERE pwsh` to make the intent more clear?

Reply:
The comment at the top of the file mentions that where does not exist in the container.

Reply:
Okay, I would have expected it in the `System32` directory. Thanks for checking!